modules/admin-system/src/domains/domains.routes.ts

```diff
@@ -0,0 +1,51 @@
+import { Routes } from '@angular/router'
+
+import { DomainListComponent } from './containers/domain-list.component'
+import { DomainDetailComponent } from './containers/domain-detail.component'
+import { DomainFormComponent } from './components/domain-form.component'
+
+import { SystemDomainResolver } from './domains.resolvers'
+
+export const SystemDomainsRoutes: Routes = [
+  {
+    path: 'domains',
+    data: { title: 'Domains' },
+    children: [
+      {
+        path: '',
+        redirectTo: 'list',
+        pathMatch: 'full',
+      },
+      {
+        path: 'list',
+        component: DomainListComponent,
+        data: { title: 'List' },
+      },
+      {
+        path: 'create',
```

Comment:
Let's keep this wrapped in the DomainDetailComponent to get better visual consistency